src/components/ui/Pool/PoolDeposit.tsx

```
@@ -472,23 +631,37 @@ const PoolDeposit = ({
+                  }
                   className="w-full mt-4"
                   onClick={
                     isConnected
                       ? () => {
-                        onDeposit(inputValueToken0, inputValueToken1);
-                        setSliderValue(50);
-                      }
+                          onDeposit(inputValueToken0, inputValueToken1);
+                          setSliderValue(50);
+                        }
                       : () => openConnectModal && openConnectModal()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)